web/src/router.ts

```diff
@@ -56,6 +57,14 @@ const routes: RouteConfig[] = [
     name: 'search',
     component: SearchView,
   },
+  {
+    path: '/dandiset/starred',
+    name: 'starredDandisets',
+    component: StarredDandisetsView,
+    meta: {
+      requiresAuth: true,
+    },
+  },
```

Comment:
This page is unviewable at the moment (at least for me), as it's lower in priority than the `'/dandiset/:identifier/:version?` URL pattern above. So when `/dandiset/starred` is navigated to, it attempts to load the dandiset with the ID of `starred`, which obviously doesn't exist. This needs to be moved above that pattern, so that it's matched first.